docs/diagrams/ArchitectureSequenceDiagram.puml

```diff
@@ -7,13 +7,13 @@ Participant ":Logic" as logic LOGIC_COLOR
 Participant ":Model" as model MODEL_COLOR
 Participant ":Storage" as storage STORAGE_COLOR
 
-user -[USER_COLOR]> ui : "delete 1"
```

Comment:
Hi, perhaps you could copy the diagram over to a folder with your name instead and restore the original file ah.